README.md

```diff
@@ -185,7 +185,7 @@ moment, the model is available under the name `Linear` in the Triton server. The
 `localhost:8000/v2/models/Linear/infer`, which are passed to the `infer_fn` function.
 
 If you would like to use Triton in the background mode, use `run`. More about that can be found
-in the [Deploying Models](https://triton-inference-server.github.io/pytriton/latest/deploying_models) page.
+in the [Deploying Models](https://triton-inference-server.github.io/pytriton/0.2.1/initialization/) page.
```

Comment:
This links also works:
https://triton-inference-server.github.io/pytriton/latest/initialization/

It can be easier to maintain the link with like this. The documentation page generates JavaScript redirect to 0.2.1 version here so it seams to be fine.

Reply:
@mahimairaja please update link in your PR to point latest alias instead of specific version as it can be easier to maintain this way. Also please update another broken deploying_models link which can be updated in this PR

Reply:
Thank you for update